### DISCUSSION TITLE
Icon Request: Step Forward/Back 10s, 15s, and 30s

### DISCUSSION THREAD

**Discussion by andyshaffer** [FortAwesome/Font-Awesome#21119] [Icon Requests]
It would be nice to have step forward/backward icons in increments of 10s, 15s, and 30s. There are other libraries that have these, and they are nice to express to users more clearly that stepping in increments is possible. 

I have an immediate need and will use custom SVG icons that I whipped up using redo-alt/undo-alt solid and Roboto Black. But these would be really helpful to have in the library with regular, light, and duo too!

![FA-Step Concepts](https://user-images.githubusercontent.com/1200694/75073506-c8ed5700-54b6-11ea-87a5-85a21677764e.png)

**Comment by jrwhip** [comment]
+1

**Comment by dalalRohit** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
We have these in FA Pro now: https://fontawesome.com/search?q=arrow-rotate&ip=classic&s=solid&c=media-playback
